benchmark/Dockerfile

```diff
@@ -27,17 +29,18 @@ RUN wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | apt
 
 COPY requirements.txt requirements.txt
 RUN python3 -m pip install --upgrade --no-cache-dir --upgrade pip && \
-  python3 -m pip install --upgrade --no-cache-dir setuptools && \
-  python3 -m pip install --upgrade --no-cache-dir -r requirements.txt && \
-  python3 -m pip install --upgrade --no-cache-dir aiomysql && \
-  python3 -m pip check
+    python3 -m pip install --upgrade --no-cache-dir setuptools && \
+    python3 -m pip install --upgrade --no-cache-dir pycairo && \
+    python3 -m pip install --upgrade --no-cache-dir -r requirements.txt && \
+    python3 -m pip install --upgrade --no-cache-dir aiomysql && \
+    python3 -m pip check
 
 ARG HAIL_WHEEL
 COPY $HAIL_WHEEL .
 RUN python3 -m pip install --upgrade --no-cache-dir --quiet $HAIL_WHEEL && \
-  python3 -m pip check
+    python3 -m pip check
```

Comment:
On a similar note to my above comment, we expect at this point that we have installed all dependencies in a previous layer, so `--upgrade` shouldn't be doing anything. I would go so far as to replace it with `--no-deps` to ensure that we aren't accidentally installing pip dependencies in this layer because that would happen every time we rebuild with changes to the hail wheel.

Reply:
smarts